src/fsharp/FSharp.Core/option.fs

```diff
@@ -31,15 +43,39 @@ namespace Microsoft.FSharp.Core
         [<CompiledName("ForAll")>]
         let forall p inp = match inp with None -> true | Some x -> p x
 
+        [<CompiledName("Contains")>]
+        let contains x inp = match inp with None -> false | Some v -> v = x
```

Comment:
That's exactly the information I was looking for. I'll add inlines appropriate to this advice.